Sources/Public/LottieConfiguration.swift

```diff
@@ -6,8 +6,12 @@
 /// Global configuration options for Lottie animations
 public struct LottieConfiguration: Hashable {
 
-  public init(renderingEngine: RenderingEngineOption = .mainThread) {
+  public init(
+    renderingEngine: RenderingEngineOption = .mainThread,
+    decodingStrategy: DecodingStrategy = .codable)
```

Comment:
I think I'm comfortable with making this the default, considering the tests pass (and the performance wins are pretty substantial!)

```suggestion
    decodingStrategy: DecodingStrategy = .dictionaryBased)
```

Reply:
I think we should let this mature a little bit just in case before making it the default. It makes me nervous to potentially break animations 🥵

Reply:
Sounds good